src/picongpu/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -71,6 +71,33 @@ AttributRadiationFlag
 
 /*########################### define species #################################*/
 
+/*--------------------------- photons -------------------------------------------*/
+
+#if(ENABLE_SYNCHROTRON_PHOTONS == 1)
+
+value_identifier(float_X, DummyMass, 1.0);
+value_identifier(float_X, DummyCharge, 0.0);
+
+typedef bmpl::vector<
+    particlePusher<particles::pusher::Photon>,
+    shape<UsedParticleShape>,
+    interpolation<UsedField2Particle>,
+    current<UsedParticleCurrentSolver>,
```

Comment:
Is the current solver called for photons as well (and just no current is returned due to  charge `=0`)?
Could you use a "None" current solver instead?